tedana/workflows/tedana.py

```diff
@@ -125,11 +125,11 @@ def _get_parser():
                         help='Perform PCA on wavelet-transformed data',
                         action='store_true',
                         default=False)
-    parser.add_argument('--label',
-                        dest='label',
+    parser.add_argument('--out',
```

Comment:
Totally subjective, but could we call it `--out-dir` to match with the string value ?

Reply:
Absolutely.